### DISCUSSION TITLE
Hey, how to please specify aliases in the commandline

### DISCUSSION THREAD

**Discussion by kubo6472** [jagrosh/MusicBot#1138] [Troubleshooting] [answered]
I tried either having it all in the quotes:
```
-Daliases="{nowplaying = [ np ]play = [ p ]remove = [ del ]stop = [ dc ]queue = [ q ]forceskip = [ fs ]skip = [ s ]movetrack = [ move ]pause = [ ps ]}"
```
or wo quotes
```
-Daliases={nowplaying = [ np ]play = [ p ]remove = [ del ]stop = [ dc ]queue = [ q ]forceskip = [ fs ]skip = [ s ]movetrack = [ move ]pause = [ ps ]}
```
first error was java treating it like a string, second some error in func. main. how to specify these correctly please? Thanks

**Comment by jagrosh** [comment, marked as answer]
First and foremost: don't do this unless you have a very good and specific reason to, because it's a mess trying to do nested objects like these. 

That being said, you'll probably need to specify values individually. For example: `-Daliases.nowplaying.0=np -Daliases.play.0=p` etc.

**Reply by kubo6472** [reply]
Oh, okay, thanks. I had issues when the bot was running on a B1s azure machine with some other software, and that the whole system was crashing. Logs from the other soft showed oom but at the same time 28mb of ram usage (the vm is 1 gig). So I had to move it elsewhere, ideally somewhere free, that's why I wanted to avoid using any files whatsoever. Not ideal, but that's what I had to do.

**Reply by jagrosh** [reply]
JMusicBot needs to be able to use files, so you will need to find another host.
